ui/scripts/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+ENV_FILENAME="/app/.env"
```

Comment:
Wouldn't be easier to add a `WORDKIR /app` statement on top of the file instead of having the `/app` dir in every command below?